lib/nunes/subscribers/action_controller.rb

```diff
@@ -12,72 +12,36 @@ def self.pattern
       end
 
       class << self
-        attr_accessor :instrument_format
         attr_accessor :instrument_view_runtime
         attr_accessor :instrument_db_runtime
       end
 
-      # Public: Should we instrument the number of requests per format overall and per controller/action.
-      self.instrument_format = true
-
       # Public: Should we instrument the view runtime overall and per controller/action.
       self.instrument_view_runtime = true
 
       # Public: Should we instrument the db runtime overall and per controller/action.
       self.instrument_db_runtime = true
 
       def process_action(start, ending, transaction_id, payload)
-        runtime = ((ending - start) * 1_000).round
-        timing "action_controller.runtime.total", runtime
-
-        status = payload[:status]
-        increment "action_controller.status.#{status}" if status
-
-        controller = ::Nunes.class_to_metric(payload[:controller])
-        action = payload[:action]
+        runtime = (ending - start) * 1_000
```

Comment:
Why remove the `.round`s?

Reply:
It will provide more precision for our metrics. Not really needed, but for any process that is happening in less than 5 milliseconds, that precision could be useful.